examples/connectivity/plot_ica_neurovault.py

```diff
@@ -0,0 +1,244 @@
+    bg_mask = compute_background_mask(src_img).get_data()
+
+    # Test if the image has been masked:
+    out_of_mask = data[np.logical_not(bg_mask)]
+    if np.all(np.isnan(out_of_mask)) or len(np.unique(out_of_mask)) == 1:
+        # Need to extrapolate
+        with warnings.catch_warnings(record=False):
+            warnings.simplefilter('ignore', RuntimeWarning)
+            data, _ = _extrapolate_out_mask(data.astype(np.float), bg_mask,
+                                            iterations=3)
+    src_img = nib.Nifti1Image(data, src_img.get_affine(),
+                              header=src_img.get_header())
+    del out_of_mask, bg_mask
+
+    # Resampling the file to target and saving the output in the "resampled"
+    # folder
+    resampled_nii = resample_img(src_img, target_img.get_affine(),
+                                 target_img.shape)
+    resampled_nii = nib.Nifti1Image(resampled_nii.get_data().squeeze(),
+                                    resampled_nii.get_affine(),
+                                    header=src_img.get_header())
+
+    # Decimate 4D files.
+    if len(resampled_nii.shape) == 3:
+        resampled_niis = [resampled_nii]
+    else:
+        resampled_niis = []
+        resampled_data = resampled_nii.get_data()
+        for index in range(resampled_nii.shape[3]):
+            # First save the files separately
+            this_nii = nib.Nifti1Image(resampled_data[..., index],
+                                       resampled_nii.get_affine())
+            resampled_niis.append(this_nii)
+
+    # Save the result
+    if out_path:
+        for nii, nii_path in zip(resampled_niis, out_paths):
+            nib.save(nii, nii_path)
+
+    return resampled_niis
+
+
+@mem.cache
+def get_neurosynth_terms(images, data_dir, print_frequency=100):
+    """ Grab terms for each image, decoded with neurosynth"""
+
+    terms = list()
+    vectorizer = DictVectorizer()
+    for ii, img in enumerate(images):
+        if ii % print_frequency == 0:
+            print("Fetching terms for images %d-%d of %d" % (
+                ii + 1, min(ii + print_frequency, len(images)), len(images)))
+
+        url = 'http://neurosynth.org/api/v2/decode/?neurovault=%d' % img['id']
+        fil = 'terms-for-image-%d.json' % img['id']
+        elevations = _fetch_files(data_dir, ((fil, url, {'move': fil}),),
+                                  verbose=2)[0]
+
+        try:
+            with io.open(elevations, 'r', encoding='utf8') as fp:
+                data = json.load(fp)['data']
+        except Exception as e:
+            if os.path.exists(elevations):
+                os.remove(elevations)
+            terms.append({})
+        else:
+            data = data['values']
+            terms.append(data)
+    X = vectorizer.fit_transform(terms).toarray()
+    return dict([(name, X[:, idx])
+                 for name, idx in vectorizer.vocabulary_.items()])
+
+
+
+# -------------------------------------------
```

Comment:
The notebook-style examples have been merged.

Once rebased on master (which I think you have just done), and you turn all these '-' into '#', you should get pretty notebook-like rendering in the docs:
https://circle-artifacts.com/gh/nilearn/nilearn/757/artifacts/0/home/ubuntu/nilearn/doc/_build/html/auto_examples/connectivity/plot_adhd_spheres.html